src/pages/settings/Payments/ChooseTransferAccountPage.js

```diff
@@ -1,27 +1,94 @@
 import React from 'react';
+import {withOnyx} from 'react-native-onyx';
+import {View} from 'react-native';
 import HeaderWithCloseButton from '../../../components/HeaderWithCloseButton';
 import ScreenWrapper from '../../../components/ScreenWrapper';
 import Navigation from '../../../libs/Navigation/Navigation';
 import withLocalize, {withLocalizePropTypes} from '../../../components/withLocalize';
 import KeyboardAvoidingView from '../../../components/KeyboardAvoidingView';
+import CONST from '../../../CONST';
+import PaymentMethodList from './PaymentMethodList';
+import * as PaymentMethods from '../../../libs/actions/PaymentMethods';
+import ROUTES from '../../../ROUTES';
+import MenuItem from '../../../components/MenuItem';
+import * as Expensicons from '../../../components/Icon/Expensicons';
+import compose from '../../../libs/compose';
+import ONYXKEYS from '../../../ONYXKEYS';
+import walletTransferPropTypes from './walletTransferPropTypes';
+import styles from '../../../styles/styles';
 
 const propTypes = {
+    /** Wallet transfer propTypes */
+    walletTransfer: walletTransferPropTypes,
+
     ...withLocalizePropTypes,
 };
 
-const ChooseTransferAccountPage = props => (
-    <ScreenWrapper>
-        <KeyboardAvoidingView>
-            <HeaderWithCloseButton
-                title={props.translate('chooseTransferAccountPage.chooseAccount')}
-                shouldShowBackButton
-                onBackButtonPress={() => Navigation.goBack()}
-                onCloseButtonPress={() => Navigation.dismissModal()}
-            />
-        </KeyboardAvoidingView>
-    </ScreenWrapper>
-);
+const defaultProps = {
+    walletTransfer: {},
+};
+
+const ChooseTransferAccountPage = (props) => {
+    /**
+     * Go back to transfer balance screen with the selected bank account set
+     * @param {Object} event Click event object
+     * @param {String} accountID of the selected account.
+     */
+    const selectAccountAndNavigateBack = (event, accountID) => {
+        PaymentMethods.saveWalletTransferAccountID(accountID);
+        Navigation.navigate(ROUTES.SETTINGS_PAYMENTS_TRANSFER_BALANCE);
+    };
+
+    /**
+     * @param {String} paymentType
+     */
+    const navigateToAddPaymentMethodPage = () => {
+        if (props.filterType === CONST.PAYMENT_METHODS.DEBIT_CARD) {
```

Comment:
Does this prop exist? I don't see it in the propTypes. Is it using the prop from PaymentMethodList?

Reply:
Ops, I forgot to refactor it. I will do it asap.